### PR TITLE
fix(condo): INFRA-425 review db setup

### PR DESCRIPTION
**Pull request by sitozzz**

### CODE REVIEW

File: .github/workflows/remove_review.yaml [hunk 1]
```diff
@@ -9,46 +9,25 @@ on:
...
+  remove_review:
     name: "Remove review"
-    needs: [check_ns]
-    if: needs.check_ns.outputs.ns_exist == 'true'
     runs-on: self-hosted
     env:
-      WERF_SECRET_KEY: ${{ secrets.WERF_SECRET_KEY }}
-      PULL_USER: ${{ secrets.PULL_USER }}
-      PULL_PERSONAL_ACCESS_KEY: ${{ secrets.PULL_PERSONAL_ACCESS_KEY }}
+      KUBE_CONFIG_BASE64_DATA: ${{ secrets.TEST_CLUSTER_KUBE_CONFIG_BASE64_DATA }}
```
Comment (SavelevMatthew) [inline .github/workflows/remove_review.yaml:16]: test?

**Review by SavelevMatthew — APPROVED** [review]

File: .github/workflows/remove_review.yaml (hunk 1 above)
Comment (sitozzz) [reply .github/workflows/remove_review.yaml:16]: I'm not a creator of that secret, so I can't edit or rename it...

**Comment by pahaz** [comment]
please check the PR title

**Comment by sitozzz** [comment]
> please check the PR title

Changed title

**Pull request merged**